docs/source/_templates/sdk_submit_tutorial.rst

```diff
@@ -153,6 +167,51 @@ SHA-512 hash of the payload bytes.
     // Check if err is nil before continuing
     transactionHeaderBytes, err := proto.Marshal(&rawTransactionHeader)
 
+{% elif language == 'Rust' %}
+
+.. code-block:: rust
+
+    extern crate protobuf;
+    extern crate openssl;
+
+    use protobuf::Message
+    use protobuf::RepeatedField;
+
+    use openssl::sha::sha512;
+    use sawtooth_sdk::messages::transaction::TransactionHeader;
+
+    let mut txn_header = TransactionHeader::new();
+    txn_header.set_family_name(String::from("intkey"));
+    txn_header.set_family_version(String::from("1.0"));
+
+    let input_vec : Vec<String>
+      = vec![String::from(
+        "1cf1266e282c41be5e4254d8820772c5518a2c5a8c0c7f7eda19594a7eb539453e1ed7")];
+    let output_vec : Vec<String>
+      = vec![String::from(
+        "1cf1266e282c41be5e4254d8820772c5518a2c5a8c0c7f7eda19594a7eb539453e1ed7")];
+
```

Comment:
Optional: Since this is a tutorial, people may refer this document to know about the APIs as well. How about a commented line at least to show how to add dependencies?

Reply:
I can totally understand that, I didn't want to add many things because It might not be the original intention. But since you agree I will address it